### ISSUE TITLE
updatedAt timestamp breaks bundle caching

### ISSUE THREAD

**Issue by thallada** [peerigon/parse-domain#60]
Every time I install the same version of this package the output changes because of the `updatedAt` timestamp changes. Unfortunately, this causes the `contenthash` to change in my webpack bundle so every time I deploy users have to re-download the whole bundle again.

Would it be possible to create some sort of hash of the results from the public suffix service instead of a timestamp? That way the built contents would be identical between installs until the suffixes change.

**Comment by jhnns** [comment]
Thank you for bringing that up. I haven't thought of this. Think we can get rid of `updatedAt` and maybe use the `fs.stat` to get the `updatedAt` information.

**Comment by overquota** [comment]
+1 for the issue. We are under PCIDSS and before we make deploy to production env we must check that hash of files inside release package and newly builded package from master are identic
updatedAt field changes every time in
```
parse-domain/build/tries/icann.complete.json
parse-domain/build/tries/icann.light.json
parse-domain/build/tries/private.complete.json
```
so its breaks our check :(

**Comment by fadedimon** [comment]
@jhnns I checked a source code and `updatedAt` is not being used anywhere. What purpose does it serve? Maybe I can send you a pull-request to remove it?

**Comment by jhnns** [comment]
🚀 Shipped with `v2.3.1`
